server/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import type Express from 'express';
```

Comment:
I think probably this file can go away too ... most of it is just delegating to endpoints at this point which can be done by `main.ts`.